internal/venice-consumer/src/main/java/com/linkedin/venice/pubsub/kafka/KafkaMessage.java

```diff
@@ -0,0 +1,50 @@
+package com.linkedin.venice.pubsub.kafka;
+
+import com.linkedin.venice.pubsub.api.PubSubMessage;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+
+
+public class KafkaMessage<K, V> implements PubSubMessage<K, V, KafkaTopicPartition, Long> {
```

Comment:
Kafka already has TopicPartition class: `org.apache.kafka.common.TopicPartition`.  Let's not add an additional wrapper unless there is something else we need out of it.

Reply:
Kafka do provide the `TopicPartition`  class, but I think `KafkaTopicPartition` extended from `PubSubTopicPartition` , `PubSubTopicPartition` with `PubSubTopic` provide more info than `TopicPartition`. Besides, we wants our `PubSub` interfaces be more general to get rid of Kafka.

Reply:
ok